flow360/__init__.py

```diff
@@ -2,6 +2,10 @@
 This module is flow360 for simulation based models
 """
 
+from examples.migration_guide.extra_operating_condition import (
```

Comment:
Let's not expose these directly under flow360 namespace. Otherwise user will see auto-prompt of these functions in IDE which is not ideal.
can you do 
```suggestion
import examples.migration_guide as migration_helper
```
and import the functions there?

Reply:
But don't we want operating_condition_from_mach_reynolds to actually be accessible normally? From what I know it is a way of describing operating conditions often used during workshop cases so I don't think it should be labeled as just migration helper.

Reply:
@maciej-flexcompute What do you think? Should we expose these function at root namespace?

Reply:
`operating_condition_from_mach_reynolds` we want to have available from flow360 namespace as this one of standard usage (workshops, airfoils etc). `muRef` should be only migration concept

Reply:
Changed it so that `operating_condition_from_mach_reynolds` is under the flow360 namespace, `operating_condition_from_muref` is available via importing from `extra_operating_condition.py` in migration_guide folder